pages/windows/clip.md

```diff
@@ -5,8 +5,16 @@
 
 - Pipe command line output to the Windows clipboard:
 
-`{{dir}} | clip`
+`dir|clip`
 
 - Copy the contents of a file to the Windows clipboard:
 
-`clip < {{path/to/file}}`
+`clip<path/to/file.ext`
+
+- Copy text with trailing newline to the Windows clipboard:
+
+`echo some text|clip`
+
+- Copy text without trailing newline to the Windows clipboard:
+
+`echo|set /p=some text|clip`
```

Comment:
Perhaps this will read better?
```suggestion
`echo | set /p={{some text}} | clip`
```

Also, is the `echo | ` required in this example? Seems a little odd to me. Don't have a windows machine to test it on.